### PR TITLE
add setting MinResources to pg for normal pod

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>
fix issue #1542 
there are two problem:
1. minresource in podgroup for deployment is empty
2. the podgroup init status is "" and volcano scheduler doesn't adjust whether the status is "", So it isgoing to cause a lot of problems. such as queue capability failure

### CODE REVIEW

File: pkg/scheduler/actions/allocate/allocate.go
Comment (shinytang6) [inline pkg/scheduler/actions/allocate/allocate.go:61]: l believe there are more code having this issue, pls help check it out.
and it's better to abstract a func to do that

**Review by shinytang6 — CHANGES_REQUESTED** [review]

File: pkg/scheduler/actions/allocate/allocate.go
Comment (huone1) [reply pkg/scheduler/actions/allocate/allocate.go]: OK, define the function "jobinfo.IsPending() bool" to check it.

File: pkg/scheduler/actions/enqueue/enqueue.go
Comment (william-wang) [inline pkg/scheduler/actions/enqueue/enqueue.go:73]: Use job.IsPending() to replace 'job.PodGroup.Status.Phase == scheduling.PodGroupPending' here.

Comment (huone1) [reply pkg/scheduler/actions/enqueue/enqueue.go:73]: don't do it, because the pending job  should be  pushed  to jobsMap.

**Review by shinytang6 — APPROVED** [review]
/lgtm

File: pkg/scheduler/actions/enqueue/enqueue.go
Comment (Thor-wl) [inline pkg/scheduler/actions/enqueue/enqueue.go:68]: Suggest to add some comments here in order not to make confusion for developers later.

File: pkg/scheduler/api/job_info.go
Comment (Thor-wl) [inline pkg/scheduler/api/job_info.go:594]: Maybe it can be merged with the `if` below.

File: pkg/scheduler/plugins/proportion/proportion.go
Comment (Thor-wl) [inline pkg/scheduler/plugins/proportion/proportion.go:270]: These logs can be removed which are just for test

Comment (huone1) [reply pkg/scheduler/plugins/proportion/proportion.go:270]: not only for test，but also for debug when the plugin behavior doesn't meet expectations. And the log level is changed to 5

File: pkg/controllers/podgroup/pg_controller_handler.go
Comment (huone1) [reply pkg/controllers/podgroup/pg_controller_handler.go:175]: the request resource of BestEffort  pod is empty  and minresource is empty, so there is no limit about the resource.

File: pkg/scheduler/actions/enqueue/enqueue.go
Comment (huone1) [reply pkg/scheduler/actions/enqueue/enqueue.go]: it  is  modified to `if job.IsPending()` with the if below

File: pkg/scheduler/api/job_info.go
Comment (huone1) [reply pkg/scheduler/api/job_info.go:594]: merged it

**Comment by jasonliu747** [comment]
/hold
@huone1 let us know when you feel ready~

**Review by shinytang6 — CHANGES_REQUESTED** [review]

File: installer/helm/chart/volcano/config/volcano-scheduler-ci.conf
Comment (huone1) [reply installer/helm/chart/volcano/config/volcano-scheduler-ci.conf:9]: plugin overcommit just  has the function jobEnqueuedFns , it shouldn't have any conflict with reclaim or preempt

**Review by Thor-wl — APPROVED** [review]

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**
